addon/components/paper-button.js

```diff
@@ -7,14 +7,22 @@ import ColorMixin from 'ember-paper/mixins/color-mixin';
 const { computed } = Ember;
 
 export default BaseFocusable.extend(RippleMixin, ProxiableMixin, ColorMixin, {
-  attributeBindings: ['type'],
   type: 'button',
```

Comment:
I believe we need to make this a computed since we don't want to render `<a type="button"`.